service/templates/cloudformation/host/route_tables.yaml

```diff
@@ -0,0 +1,17 @@
+{{define "route_tables"}}
+  {{ range .RouteTables }}
+  PrivateRouteTable-{{.Name}}:
+    Type: AWS::EC2::RouteTable
+    Properties:
+      VpcId: {{.PeerVPCID}}
+      Tags:
+      - Key: Name
+        Value: {{ .Name }}
```

Comment:
The route tables aren't needed. They are created in the host cluster setup. We just need to create the routes.